types/block_meta.go

```diff
@@ -1,16 +1,18 @@
 package types
 
-// BlockMeta contains meta information about a block - namely, it's ID and Header.
+// BlockMeta contains meta information about a block - ID, Size, and Header.
 type BlockMeta struct {
-	BlockID BlockID `json:"block_id"` // the block hash and partsethash
-	Header  Header  `json:"header"`   // The block's Header
+	BlockID   BlockID `json:"block_id"`   // the block hash and partsethash
+	BlockSize int     `json:"block_size"` // The size of the block
```

Comment:
I wonder if we should make this `uint32` or simply `uint`

Reply:
I am for this change, there isn't a case where this should ever be -, uint seems good enough